pkg/backup/safe_point.go

```diff
@@ -4,22 +4,27 @@ package backup
 
 import (
 	"context"
+	"fmt"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
 	pd "github.com/pingcap/pd/v4/client"
 	"go.uber.org/zap"
 )
 
 const (
-	brServiceSafePointID            = "br"
+	brServiceSafePointIDFormat      = "br-%s"
 	preUpdateServiceSafePointFactor = 3
 	checkGCSafePointGapTime         = 5 * time.Second
 	// DefaultBRGCSafePointTTL means PD keep safePoint limit at least 5min
 	DefaultBRGCSafePointTTL = 5 * 60
 )
 
+// for each BR, use different safe point ID so they won't conflict.
+var brServiceSafePointID = fmt.Sprintf(brServiceSafePointIDFormat, uuid.New())
```

Comment:
Please do not use global variable, TiDB integrates BR and it could execute multiple backups at the same time.

Reply:
OK, I think we can inject it at `RunBackup`.